controller/test/test_refugee_controller.py

```diff
@@ -0,0 +1,164 @@
+
+    def test_find_refugee_return_type(self):
+        """
+        Test case where refugee specified is returned as Refugee type.
+        """
+        Plan(name='test_plan1',
+             emergency_type=Plan.EmergencyType.EARTHQUAKE,
+             description='Test emergency plan',
+             geographical_area='London',
+             camps=[Camp(name='camp1')])
+        test_plan = Plan.find('test_plan1')
+        test_camp = test_plan.camps.get('camp1')
+        refugee1 = Refugee(firstname="Tom",
+                           lastname="Bond",
+                           num_of_family_member=600,
+                           starting_date=date(2020, 1, 2),
+                           medical_condition_type=[Refugee.MedicalCondition.HIV, Refugee.MedicalCondition.CANCER])
+        test_camp.refugees.add(refugee1)
+        refugee_user_id = refugee1.user_id
+        refugee = find_refugee(refugee_user_id)
+        self.assertEqual(type(refugee), Refugee)
+
+    def test_find_refugee_return_no_refugee_found(self):
+        """
+        Test case where refugee specified is found in the file.
+        """
+        Plan(name='test_plan1',
+             emergency_type=Plan.EmergencyType.EARTHQUAKE,
+             description='Test emergency plan',
+             geographical_area='London',
+             camps=[Camp(name='camp1')])
+        test_plan = Plan.find('test_plan1')
+        test_camp = test_plan.camps.get('camp1')
+        refugee1 = Refugee(firstname="Tom",
+                           lastname="Bond",
+                           num_of_family_member=600,
+                           starting_date=date(2020, 1, 2),
+                           medical_condition_type=[Refugee.MedicalCondition.HIV, Refugee.MedicalCondition.CANCER])
+        test_camp.refugees.add(refugee1)
+        refugee_user_id = 1111111111
+        refugee = find_refugee(refugee_user_id)
+        self.assertIsNone(refugee, refugee1)
+
+class ViewRefugeeTest(unittest.TestCase):
+    """
+    Class for testing cases in view refugee function.
+    """
+    def test_view_refugee_return_value(self):
+        """
+        Test case where refugee view is same as the refugee specified.
+        """
+        Plan(name='test_plan1',
+             emergency_type=Plan.EmergencyType.EARTHQUAKE,
+             description='Test emergency plan',
+             geographical_area='London',
+             camps=[Camp(name='camp1')])
+        test_plan = Plan.find('test_plan1')
+        test_camp = test_plan.camps.get('camp1')
+        refugee1 = Refugee(firstname="Terry",
+                           lastname="Bimble",
+                           num_of_family_member=2,
+                           starting_date=date(2020, 1, 2),
+                           medical_condition_type=[Refugee.MedicalCondition.HIV, Refugee.MedicalCondition.CANCER])
+        test_camp.refugees.add(refugee1)
+        refugee_user_id = refugee1.user_id
+        refugee = find_refugee(refugee_user_id)
```

Comment:
For atomicity of tests, i suggest avoiding find_refugee here
Just view the created instance